src/lipalightninglib.udl

```diff
@@ -385,6 +389,24 @@ dictionary FiatTopupInfo {
     string creditor_country;
 };
 
+// Information on a funds offer that can be claimed using a kind-dependent method (e.g. LNURL-w).
+dictionary OfferInfo {
+    OfferKind offer_kind;
+    Amount amount;
+    timestamp expires_at;
+};
+
+[Enum]
+interface OfferKind {
+    Pocket(
+        string lnurl,
+        u64 exchange_fee_fiat_minor_unit,
+        string exchange_fee_fiat_currency,
+        f64 exchange_fee_rate
```

Comment:
Makes sense! I initially avoided that because the source of data is completely different. But it makes sense to adapt it to our model. Applied in [00e0b3a](https://github.com/getlipa/lipa-lightning-lib/pull/505/commits/00e0b3a768f003df7b8c0adc7426d479614c2663)